pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -161,6 +161,13 @@ func (c *ClusterConfig) ValidateVPCConfig() error {
 		}
 		c.VPC.PublicAccessCIDRs = cidrs
 	}
+	if len(c.VPC.ExtraIPv6CIDRs) > 0 {
```

Comment:
a user should only be able to set `vpc.extraIPv6CIDRs` if they've also set `vpc.IpFamily` to `IPv6`

Reply:
eeeek good spot!

Reply:
done!